File: src/main/java/com/blackcowmoo/moomark/auth/model/dto/TokenResponse.java
package com.blackcowmoo.moomark.auth.model.dto;

import com.blackcowmoo.moomark.auth.model.AuthProvider;
import com.blackcowmoo.moomark.auth.model.Role;

import lombok.Data;

@Data
public class TokenResponse {
  public String id;
  public AuthProvider provider;
  public Role role;
}

File: src/main/kotlin/com/blackcowmoo/moomark/auth/model/dto/TokenResponse.kt
package com.blackcowmoo.moomark.auth.model.dto

import com.blackcowmoo.moomark.auth.model.AuthProvider
import com.blackcowmoo.moomark.auth.model.Role
import lombok.Data

@Data
class TokenResponse(
  var id: String? = null,
  var provider: AuthProvider? = null,
  var role: Role? = null
)
